packages/file-format/scripts/validate-file.ts

```diff
@@ -3,43 +3,12 @@ import { ErrorObject } from 'ajv'
 import util from 'util'
 
 import { getAjvInstance } from './utils'
-import schema from './file-format.schema.json'
-
-// TODO: This `fromFile` (and presumably a `toFile`) function should be
-// published publically in the future
-const fromFile = async (filepath: string): Promise<any> => {
-  const archive = new StreamZip({
-    file: filepath,
-    storeEntries: true,
-  })
-
-  const contents = await new Promise((resolve): void => {
-    archive.on('ready', (): void => {
-      const document = JSON.parse(
-        archive.entryDataSync('document.json').toString(),
-      )
-      const pages = document.pages.map((page: { _ref: string }): void =>
-        JSON.parse(archive.entryDataSync(`${page._ref}.json`).toString()),
-      )
-      resolve({
-        document: {
-          ...document,
-          pages,
-        },
-        meta: JSON.parse(archive.entryDataSync('meta.json').toString()),
-        user: JSON.parse(archive.entryDataSync('user.json').toString()),
-      })
-    })
-  })
-
-  archive.close()
-
-  return contents
-}
+import schema from '../src/file-format.schema.json'
+import { fromFile, SketchFile } from '../../file'
```

Comment:
The beauty of the monorepo is that instead of relative imports you can treat `@sketch-hq/sketch-file` just like a normal dependency.

```suggestion
import { fromFile, SketchFile } from '@sketch-hq/sketch-file'
```

It would have to be added to the [`package.json`](https://github.com/sketch-hq/sketch-document/blob/50285d383ea4065a2b2f1df30eb0360823f3d914/packages/file-format/package.json) just like in the TypeScript types:
 
https://github.com/sketch-hq/sketch-document/blob/50285d383ea4065a2b2f1df30eb0360823f3d914/packages/file-format-ts/package.json#L23

Reply:
I tried to do that but for some reason it would not work. Let me try again...

Reply:
Ah, looks like I was missing the `package.json` when I tried, works now!